manticore/core/state.py

```diff
@@ -231,6 +231,19 @@ def solve_n(self, expr, nsolves, policy='minmax'):
         '''
         return self._solver.get_all_values(self.constraints, expr, nsolves, silent=True)
 
+    def solve_buffer(self, addr, nbytes):
+        '''
+        Reads `nbytes` of symbolic data from a buffer in memory at `addr` and attempts to
+        concretize it
+
+        :param int address: Address of buffer to conretize
+        :param int nbytes: Size of buffer to conretize
+        :return: Concrete contents of buffer
+        :rtype: list[int]
+        '''
+        buffer = self.cpu.read_bytes(addr, nbytes)
+        return ''.join(chr(self.solve_one(x)) for x in buffer)
```

Comment:
array[0] = bitvec(8)
array[1] = bitvec(8)
solver.add_assertion(array[0] >array[1])
solver.get_value(array[0]) - > 0
solver.get_value(array[1]) - > 0

->  := "it could give"